modules/ROOT/pages/business-events.adoc

```diff
@@ -13,32 +13,31 @@ Mule applications collect _business event_ information (such as transaction exec
+See xref:business-events-in-components.adoc[] for configuration details.
 
-Custom event tracking is supported by the Custom Business Event component. Custom events are always tracked, enabling you to monitor high-level activities in your flow that are relevant to your business. +
-See xref:business-events-custom.adoc[Custom Business Event Component] for configuration details.
+Custom event tracking is supported by the Custom Business Event component. Custom events are always tracked, enabling you to monitor high-level activities in your flow that are relevant to your business. See xref:business-events-custom.adoc[] for configuration details.
 
 == Monitoring Business Events
 
 You can enable Insight in Runtime Manager to monitor business events at runtime. These events can help you analyze the root cause of failures, isolate performance bottlenecks, and test for compliance with company procedures.
 
 Certain default events are tracked automatically after you enable Insight, but you must enable others in your application before they are tracked.
 
-See the xref:runtime-manager::insight.adoc[Insight] documentation for more details.
+See xref:runtime-manager::insight.adoc[] for more details.
 
 == Best Practices for Business Events
 
 Consider the following recommended practices when configuring business events tracking:
 
 * Enable default events only for processes that have particular value to you. To filter out unnecessary information, determine which stages within a business transaction you need to track, and enable tracking for only those stages.
-* Use custom events to track key process indicators: for example, "Total Order Amount" or "Tracking Number" to surface the high-level business activities in your flow.
-* Customize the transaction ID so that meaningful information, such as an order number, an employee identification number, or a shipment tracking number, is displayed for a transaction. +
+* Use custom events to track key process indicators, for example, "Total Order Amount" or "Tracking Number" to surface the high-level business activities in your flow.
+* Customize the transaction ID with xref:set-transaction-id.adoc[Set Transaction Id] so that meaningful information, such as an order number, an employee identification number, or a shipment tracking number, is displayed for a transaction.
 This practice makes analysis and debugging easier and more intuitive at runtime.
 
 == See Also
 
-* xref:about-mule-event.adoc[Mule Events]
-* xref:transaction-management.adoc[Transaction Management]
-* xref:business-events-in-components[Configure Default Events Tracking]
-* xref:business-events-custom.adoc[Custom Business Event Component]
+* xref:about-mule-event.adoc[]
+* xref:transaction-management.adoc[]
```

Comment:
TODO: In the intro, say something about the relationship to business events and transaction management. Doc needs more context in this regard.